src/components/Book.css

```diff
@@ -1,25 +1,187 @@
 .bks_main_cont {
-  width: 100%;
+  display: flex;
+  background-color: #fff;
+  width: 90%;
+  height: 100%;
+  margin: 0 auto;
+  margin-top: 20px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
 }
 
 .bks_main_cont .books_container {
   display: flex;
   flex-direction: column;
   justify-content: center;
-  width: 85%;
   margin: 0 auto;
   margin-top: 4%;
   margin-bottom: 2%;
   padding-left: 2%;
   align-items: center;
+  width: 40%;
   height: 100px;
+}
+
+.bks_main_cont .books_container .category_tl {
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
```

Comment:
- [ ] The default value for `font-stretch`, `font-style`, `line-height`, `letter-spacing` is "normal", so you don't have to write these CSS properties. To follow the best practice and for Clean CSS code 💯